test/testdata/lsp/symbol_query_filter__main.rb

```diff
@@ -16,7 +16,7 @@ def isFoo
   end
 end
 
-class SomeClass
+class SomeClass # error: The super class `AnotherModule::NestedModule` of `SomeClass` does not derive from `Class`
```

Comment:
Extremely slight bummer that this error locates here when the actual offense is down below when we reopen the class. (But I guess one sees it down there as well, and I dunno if there's any way to make this not happen.)

Reply:
Good point. I've added an additional check to exit early if the `classDef` is missing a superclass 👍